### ISSUE TITLE
Would it make sense to mention/list rubocop extensions

### ISSUE THREAD

**Issue by vfonic** [rubocop-hq/rubocop#7133]
Maybe as one of the sections in the Readme.md?

## Describe the solution you'd like

If you're new to rubocop, it's hard to know and find existing rubocop extensions. You'd have to go to `rubocop-hq` and check which repos are there, etc. We could have the most popular or all extensions right in the Readme.md to avoid confusion and people not finding extensions, especially the ones like rubocop-rails or rubocop-rspec.

What do you think?

**Comment by pocke** [comment]
You can find popular custom cops from here https://docs.rubocop.org/en/stable/extensions/#custom-cops

Personally, I think README does not need to mention the custom cops because it already has a link to the official manual.
But, I think we should improve the README and official manual if people are confused by them. For example, more eye-catching link to the manual, add useful ToC  to the top page of the manual, etc.

**Comment by bbatsov** [comment]
Yeah, I agree. Having everything important in the manual is what matters the most IMO. 

> But, I think we should improve the README and official manual if people are confused by them. For example, more eye-catching link to the manual, add useful ToC to the top page of the manual, etc.

I totally agree here as well.

**Comment by vfonic** [comment]
// I wrote a lengthy reply and then removed it 😇 

tl;dr
Perhaps "Official manual" could be just changed to "Documentation"?

I just opened couple of repos that I could remember from the top of my head and here's the terminology they use (this is random sample):
https://github.com/facebook/create-react-app/blob/master/README.md#user-guide (words used: "User Guide" and "documentation)
https://github.com/bumps/bumps#bumps-data-fitting-and-uncertainty-estimation (words used: "documentation")
https://github.com/avajs/ava#documentation (words used: "documentation")
https://github.com/thoughtbot/factory_bot#documentation (words used: "documentation")
https://github.com/norman/friendly_id#friendlyid (words used: "documentation")

I also really like how FriendlyId does it. It's the first sentence in the README:
https://github.com/norman/friendly_id

PS what about these two extensions:
* https://github.com/rubocop-hq/rubocop-md
* https://github.com/rubocop-hq/rubocop-sequel
They don't seem to be in the list of extensions in the documentation.

**Comment by bbatsov** [comment]
Fair point! I guess I meant to write official User Manual or something like this back in the day, but mistyped or whatever. :D I'm fine with rebranding to "User Manual", "documentation" or whatever else people prefer. 

> They don't seem to be in the list of extensions in the documentation.

An oversight we should address.
